### PR TITLE
Fix/new-review getUser()

### PR DESCRIPTION
**Pull request by craigostrin**

### CODE REVIEW

File: apps/client/app/employee/[id]/new-review/page.tsx
Comment (bienmag) [inline apps/client/app/employee/[id]/new-review/page.tsx:1]: it's already in main

File: apps/client/app/employee/[id]/new-review/page.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const reportQuery = `
 const panelPadding = 'p-4'
 
 export default async function Review({ params }: { params: any }) {
-  const user = await getUser(params.id, userQuery)
+  const user = await getUser(userQuery, { id: params.id })
```
Comment (bienmag) [inline apps/client/app/employee/[id]/new-review/page.tsx:46]: it's already in main

**Review by bienmag — CHANGES_REQUESTED** [review]

**Comment by bienmag** [comment]
what should we do with this?

**Comment by craigostrin** [comment]
outdated

**Pull request closed without merging**